Gemfile

```diff
@@ -56,6 +56,9 @@ gem "libxml-ruby", platforms: :ruby
 # Action View. For testing Erubis handler deprecation.
 gem "erubis", "~> 2.7.0", require: false
 
+# for railties app_generator_test
+gem "bootsnap", ">= 1.1.0", require: false
```

Comment:
The application inside the railties test suite uses the Rails main Gemfile. I don't see a problem on adding bootsnap in the main gemfile, we already do to gems like jbuilder, erubis, etc.